jsonnetlib/k8s.jsonnet

```diff
@@ -314,6 +314,7 @@ cfg {
     env=[],
     container={},
     volumes=[],
+    terminationGracePeriodSeconds=30,
```

Comment:
```suggestion
    terminationGracePeriodSeconds=root.terminationGracePeriodSeconds,
```

Reply:
please move the default value to https://github.com/theplant/plantbuild/blob/master/jsonnetlib/config.jsonnet

Reply:
Thanks, moved to config.jsonnet.